### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by toprak**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -85,6 +86,7 @@ A curated list of anime streaming/downloading websites.
 
  * [animebytes.tv](https://animebytes.tv/)
  * [animetorrents.me](https://animetorrents.me/)
+ * [bakabt.me](https://bakabt.me)
```
Comment (anshumanv) [inline README.md:89]: Also please explain how someone can get an account here :)

File: README.md [hunk 2]
```diff
@@ -65,14 +65,15 @@ A curated list of anime streaming/downloading websites.
...
+* [turkanime.net](https://turkanime.net/) - Contains Turkish Subs
+* [tranimeizle.net](https://tranimeizle.net) - Contains Turkish Subs
 
 ### Anime (mobile app)
 * [#Animania](http://www.mobi24.net/#anime)
 
 ### Torrents
 * [animetosho.org](https://animetosho.org) - Torrent, NZB, DDL
 * [anidex.info](https://anidex.info/)
-* [animetorrents.me](https://animetorrents.me/)
```
Comment (anshumanv) [inline README.md:75]: why remove?

File: README.md [hunk 3]
```diff
@@ -65,14 +65,15 @@ A curated list of anime streaming/downloading websites.
 * [station.f-n.me](https://station.f-n.me/)
 * [thehylia.com](https://anime.thehylia.com/)
 * [watchcartoononline.io](https://www.watchcartoononline.io/)
+* [turkanime.net](https://turkanime.net/) - Contains Turkish Subs
+* [tranimeizle.net](https://tranimeizle.net) - Contains Turkish Subs
```
Comment (anshumanv) [inline README.md:69]: this doesn't seem to load for me, does it work for you?

**Review by anshumanv — CHANGES_REQUESTED** [review]
Thanks for the contributions, please check comments

File: README.md (hunk 3 above)
Comment (toprak) [reply README.md:69]: both of them are working

File: README.md (hunk 2 above)
Comment (toprak) [reply README.md:75]: I did because you already created a section for private trackers right?

Comment (anshumanv) [reply README.md:75]: ah okay yeah this seems good

File: README.md (hunk 1 above)
Comment (toprak) [reply README.md:89]: If you want to get a bakabt account, you should go to their irc which is #BakaBT on Rizon, and take a queue number for getting reviewed.

Comment (anshumanv) [reply README.md:89]: can you just add this message besides the link, see how others are done :)

Comment (toprak) [reply README.md]: i suppose its okay now

**Review by anshumanv — APPROVED** [review]
Thanks ✨

**Pull request merged**